recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
> Both attempts failed on OS X. BTW, have you noticed that the issue is on make check step after the library itself is built successfully?

Thanks @frol. Sorry, I hadn't had a chance to look closer.

This should be an easy fix. The problem is related to how dynamic libraries are loaded. Will update the diff with some comments that should get this moving again. :smile: